### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by go4ble**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/go4ble/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 this is the readme
+
+this is the readme with a pending change
```
Comment (go4ble) [inline README.md]: is this all you're changing!?

Comment (go4ble) [inline README.md]: add some punctuation! !fix

Comment (go4ble) [inline README.md]: fixed it !fixed

**Comment by go4ble** [comment]
!star

**Pull request merged**